README.md

```diff
@@ -52,124 +54,66 @@ Add the following Maven project to your `pom.xml`:
 </dependency>
 ```
 
+#### Gradle
+
+```kotlin
+implementation("io.github.cdklabs:cdkmonitoringconstructs:0.0.11")
+```
+
 ### Python (PyPi)
 
-> Repository: https://pypi.org/project/cdk-monitoring-constructs/
+> https://pypi.org/project/cdk-monitoring-constructs/
 
 TODO: describe usage
 
 ### C# (Nuget)
 
-> Repository: https://www.nuget.org/packages/Cdklabs.CdkMonitoringConstructs/
+> https://www.nuget.org/packages/Cdklabs.CdkMonitoringConstructs/
 
 TODO: describe usage
 
-## Documentation
+## Features
 
 See [API](API.md) for complete auto-generated documentation.
 
+You can also browse the documentation at https://constructs.dev/packages/cdk-monitoring-constructs/
+
+| Item | Monitoring | Alarms | Notes |
```

Comment:
This is a great way to represent the features!

Small nitpick, is it too much to ask for the items currently mentioned in braces to be in bullets? (Not sure how the table markdown plays with it honestly)

Reply:
We'd need to use some vanilla HTML for that. Are you talking about stuff like the "`.monitorApiGateway()`"?

Reply:
Yep those stuff are getting a bit un-readable for services with many aspects. But let's deal with it later when we want to improve our docs further.